src/Uno.UI/WasmCSS/Uno.UI.css

```diff
@@ -8,6 +8,15 @@ body {
   width: 100%;
   overflow: hidden;
   -webkit-tap-highlight-color: rgba(0,0,0,0);
+
+  /*
+    Disable **browsers** touche (and pen) manipulations support.
+    Remarks: This has no relation with the UIElement.ManipualtionMode which are handled in managed code.
+             It only ensure that uno will always get ALL pointers events instead of being stollen by the browser for its internal gesture detection.
```

Comment:
```suggestion
             It only ensures that uno will always get ALL pointer events instead of being stolen by the browser for its internal gesture detection.
```